### PR TITLE
Adds helpers for working with prompts

### PR DESCRIPTION
**Pull request by jaresty**
- Sometimes it's useful to set the prompt to your clipboard or view in the browser

### CODE REVIEW

File: GPT/gpt.talon [hunk 1]
```diff
@@ -10,8 +10,8 @@ model <user.modelPrompt> [{user.modelSource}] [{user.modelDestination}]:
     result = user.gpt_apply_prompt(modelPrompt, text)
     user.gpt_insert_response(result, modelDestination or "")
 
-# Select the last GPT response so you can edit it further
-model take response: user.gpt_select_last()
```
Comment (C-Loftus) [inline GPT/gpt.talon:14]: We are getting rid of this? This doesn't seem contradictory to the new addition

Comment (jaresty) [reply GPT/gpt.talon:14]: Oops. That looks like a mistake. I'm afk but will put that back if you don't beat me to it.

Comment (jaresty) [reply GPT/gpt.talon:14]: Reverted that accidental deletion

**Comment by C-Loftus** [comment]
Ok  I am fine with this now. I put this in gpt-beta since I am thinking this might be condensed into the grammar at some point but not sure what that will exactly look like now. I am using `model pass <prompt>` so it fits the syntax better. `model prompt` is a bit confusing imo. `model pass` signifies we are doing a noop where the source is the prompt itself.

Added some comments. Let's try to always get comments on the talon files whenever we add something.

**Comment by C-Loftus** [comment]
Also moves the code that adds the snip modifier prompt _before_ the code that does the passing. This doesn't change any behavior atm, but in general as a best practice we should be checking special cases _after_ the prompt is finalized by checking modifiers.

**Comment by jaresty** [comment]
Looks good to me!

**Pull request merged**